ui/pages/multichain-accounts/multichain-account-details-page/multichain-account-details-page.tsx

```diff
@@ -102,6 +105,11 @@ export const MultichainAccountDetailsPage = () => {
                 ariaLabel={t('addresses')}
                 marginLeft={2}
                 data-testid="network-addresses-link"
+                onClick={() => {
```

Comment:
the web developer in me wishes this was a simple `href` :) 

I think ButtonIcon has `as` prop?

Reply:
Great catch! I'll updated in the UI polish phase. Issue here https://github.com/MetaMask/metamask-extension/issues/35533